### PR TITLE
A0-1855: In FE, replace private feature-env-aleph-node repo with public aleph-node

### PR DESCRIPTION
**Pull request by mikogs**
All feature env docker images should now use the public aleph-node repository, instead of the private one.

### CODE REVIEW

**Comment by mikogs** [comment]
I have created `feature-env-aleph-node` public ECR repository, with URL of `public.ecr.aws/p6e8q1z1/feature-env-aleph-node`

File: .github/workflows/deploy-feature-envs.yaml [hunk 1]
```diff
@@ -11,9 +11,9 @@ env:
   LABEL_DESTROYED: 'DESTROYED'
   LABEL_DEPLOYED: 'DEPLOYED'
   LABEL_DEPLOYED_CONTRACTS: 'DEPLOYED-CONTRACTS'
-  REGISTRY_HOST: 573243519133.dkr.ecr.us-east-1.amazonaws.com
-  FE_ALEPHNODE_REGISTRY: 573243519133.dkr.ecr.us-east-1.amazonaws.com/feature-env-aleph-node
-  FE_ALEPHNODE_REGISTRY_ESCAPED: '573243519133.dkr.ecr.us-east-1.amazonaws.com\/feature-env-aleph-node'
+  REGISTRY_HOST: public.ecr.aws
+  FE_ALEPHNODE_REGISTRY: public.ecr.aws/p6e8q1z1/feature-env-aleph-node
+  FE_ALEPHNODE_REGISTRY_ESCAPED: 'public.ecr.aws\/p6e8q1z1\/feature-env-aleph-node'
```
Comment (fbielejec) [inline .github/workflows/deploy-feature-envs.yaml:16]: what does this puppy do?

Comment (mikogs) [reply .github/workflows/deploy-feature-envs.yaml:16]: So, it is used in one of the steps:
````
- name: Update feature environment to the latest PR image
        env:
          (...)
        run: |
          # Set up envs
          PR_IMAGE="${{ env.FE_ALEPHNODE_REGISTRY_ESCAPED }}\:${{ env.IMAGE_TAG }}"
          (...)
          sed "s/NODE_IMAGE/$PR_IMAGE/g" $ALEPH_PATH/aleph-apps/argocd/overlays/devnet/fe-apps/${{ env.APP_NAME }}.yaml > $ALEPH_PATH/aleph-apps/argocd/overlays/devnet/fe-apps/${{ env.APP_NAME }}.yaml.temp && mv $ALEPH_PATH/aleph-apps/argocd/overlays/devnet/fe-apps/${{ env.APP_NAME }}.yaml.temp $ALEPH_PATH/aleph-apps/argocd/overlays/devnet/fe-apps/${{ env.APP_NAME }}.yaml
````
For some reason, this variable needs to have slashes escaped.  It's used in `sed`.  I remember trying to remove that nasty thing with by taking the `FE_ALEPHNODE_REGISTRY` and just replacing `/` with `\/` but that was problematic.  Hence that ugly additional variable is added.

**Review by fbielejec — APPROVED** [review]
looks good!

**Review by Kosiare — APPROVED** [review]

**Pull request merged**